scripts/custom/aica_bridge/run_bridge.py

```diff
@@ -226,8 +226,8 @@ def main() -> None:
         address=arguments.ip_address,
         state_port=arguments.state_port,
         command_port=arguments.command_port,
-        force_port=arguments.force_port,
-        force_sensor_name=arguments.force_sensor,
+        force_port=arguments.ft_sensor_port,
```

Comment:
As long as we're breaking stuff, you don't want to propagate that to the bridge?